### PR TITLE
New package: pipe-viewer-0.0.7

### PR DESCRIPTION
**Pull request by 4ricci**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ericonr** [comment]
We already ship `youtube-viewer`, `straw-viewer` and a few others. Do you think this one is noticeably different?

**Comment by 4ricci** [comment]
@ericonr see #29015

File: srcpkgs/pipe-viewer/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+short_desc="Search and play videos from YouTube"
+maintainer="Roberto Ricci <ricci@disroot.org>"
+license="Artistic-2.0"
+homepage="https://github.com/trizen/pipe-viewer"
+changelog="https://github.com/trizen/pipe-viewer/releases"
+distfiles="https://github.com/trizen/pipe-viewer/archive/${version}.tar.gz"
+checksum=5e1211ec963aa2a187bcb0983227c5667e6c580bfc9de93dcb582388cc1baf22
+
+pipe-viewer-gtk_package() {
+	depends="${sourcepkg}-${version}_${revision} perl-Gtk3 perl-File-ShareDir"
```
Comment (ericonr) [inline srcpkgs/pipe-viewer/template:20]: This should be `sourcepkg>=`

**Comment by ericonr** [comment]
Interesting, thanks for the context!

Once straw-viewer is deprecated, do you think pipe-viewer could receive a meta package for it?

**Comment by 4ricci** [comment]
Yes, I could add a dummy subpackage to pipe-viewer.

**Pull request merged**